### PR TITLE
feat: mobile assettable

### LINKED ISSUE #497: Add current value in single stock is asset list
Currently only price per share and current price are shown. I believe the user is especially interested in seeing the total current value of the asset so, current price x quantity owned. I know this is also displayed within the weight, however i think it would be nice to name it explicitly maybe when hovering over the weight or simply as a text next to the current price

### PR DESCRIPTION
**Pull request by SaitaoGit**
close #497

### CODE REVIEW

**Comment by chronark** [comment]
#### 📝 Changed routes:


| [**/portfolio/\[portfolioId\]**](https://perfolio-f1cuc1i8x-perfolio.vercel.app/portfolio/[portfolioId]) |
|:-:|
|<a href="https://perfolio-f1cuc1i8x-perfolio.vercel.app/portfolio/[portfolioId]">Screenshot of /portfolio/[portfolioId]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/w-0U3WS30YManaGZaCJV8VZkM5DvrunbDK6lOPveEujAOBNyAqHg-Hho3349PhlwThJUT4W1vK8XX38HGzdD-g.png">(view full size)</a> |







Commit a56922aea85f0e40d97138e5cbecf675d128b785 (https://perfolio-f1cuc1i8x-perfolio.vercel.app).

File: ui/app/assetTable/assetTable.tsx [hunk 1]
```diff
@@ -47,125 +49,208 @@ export const AssetTable: React.FC<AssetTableProps> = ({
...
+                    {format(holding.value * holding.quantity, { suffix: "€" })}
+                  </Cell.Text>
+                ),
+                change: (
+                  <Cell.Tag
+                    align="text-right"
+                    textColor={change > 0 ? "text-success-dark" : "text-error-dark"}
+                    bgColor={change > 0 ? "bg-success-light" : "bg-error-light"}
+                  >
+                    <span onClick={() => setDisplayAggregation(displayAggregation === "absolute" ? "relative" : "absolute")}>
```
[the remark this answers is not in the document]
Comment (SaitaoGit) [reply ui/app/assetTable/assetTable.tsx]: done

File: ui/app/assetTable/assetTable.tsx [hunk 2]
```diff
@@ -47,125 +49,208 @@ export const AssetTable: React.FC<AssetTableProps> = ({
...
+                          </div>
+                        </div>
+                      </div>
+                      <div
+                        className={cn(
+                          change > 0 ? "text-success-dark" : "text-error-dark",
+                          change > 0 ? "bg-success-light" : "bg-error-light",
+                          "self-center px-1 rounded",
+                        )}
+                        onClick={() => setDisplayAggregation(displayAggregation === "absolute" ? "relative" : "absolute")}
```
[the remark this answers is not in the document]
Comment (SaitaoGit) [reply ui/app/assetTable/assetTable.tsx]: done

File: ui/app/main/content.tsx [hunk 3]
```diff
@@ -4,5 +4,5 @@ import React from "react"
 export interface MainContentProps {}
 
 export const MainContent: React.FC<MainContentProps> = ({ children }): JSX.Element => {
-  return <div className="p-4 rounded-b md:p-6">{children}</div>
+  return <div className="sm:p-4 rounded-b md:p-6">{children}</div>
```
[the remark this answers is not in the document]
Comment (SaitaoGit) [reply ui/app/main/content.tsx:7]: Actually we would have less place than. 
At the moment we have no padding at all at screens smaller than sm

[the remark this answers is not in the document]
Comment (chronark) [reply ui/app/main/content.tsx:7]: Fair enough, my mistake

**Pull request merged**